src/Renderer/ThreeExtended/FeaturesUtils.js

```diff
@@ -0,0 +1,114 @@
+function overLine(point, vs, precision) {
+    const x0 = point._values[0];
+    const y0 = point._values[1];
+    for (var i = 0, j = vs.length - 1; i < vs.length; j = i++) {
```

Comment:
This funny for-loop construct is surely justified but please add a comment to explain the intent.